docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -258,24 +272,27 @@ public class Startup {
 }
 ----
 
-The application is now protected and the identities are provided by our database.
+The application is now protected and the user identities are provided by the specified database.
 
 [NOTE]
 ====
-We kindly remind you that you must not store clear-text passwords in production environments ;-).
+In a production environment, do not store plain text passwords.
 As a result, the `security-jpa` defaults to using bcrypt-hashed passwords.
 ====
 
-== Testing the Application
+== Test your application
 
-=== With Dev Services for PostgreSQL
+=== Test your application by using Dev Services for PostgreSQL
```

Comment:
```suggestion
=== Use Dev Services for PostgreSQL to test your application
```

Given this is just below the `Test your application` heading, seems like we can still be direct and highlight what is unique about this section

Reply:
We ususally would frown upon 'use' or 'using' as the first word verb in an action heading as they can be ambiguous and too broad.

But... as you rightly point out, this heading is directly under the Test your application, so I think that's fine in this case, unless it doesn't cause problems for the other children (===) in this section that should be in parallel.